presentation-api/controlling-ua/startNewPresentation_success-manual.html

```diff
@@ -31,8 +32,10 @@
             return request.start()
                     .then(function (connection) {
 
-                        // assert case for the promise_test
-                        assert_equals(connection.state, "connected", "The presentation has an connected state.");
+                        // Check the initial state of a presentation connection
+                        test(function() {
```

Comment:
Similarly, I would rewrite this test with an enclosing `async_test` call and no `test` calls to ensure consistency between a PASS and a FAIL outcome.